app/test/e2e/webshot-screenshot-post.spec.js

```diff
@@ -0,0 +1,81 @@
+const chai = require('chai');
+const nock = require('nock');
+const sinon = require('sinon');
+
+const { getTestServer } = require('./utils/test-server');
+const { stubPuppeteer, stubKoaSend } = require('./utils/stubs');
+
+chai.should();
+
+let requester;
+let sinonSandbox;
+
+nock.disableNetConnect();
+nock.enableNetConnect(process.env.HOST_IP);
+
+describe('GET /', () => {
```

Comment:
please use better descriptions for the test cases. I know this MS only has a single GET endpoint, but that doesn't make this a good naming policy